src/app.js

```diff
@@ -1,21 +1,48 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const staticFiles = () => {
+  const server = http.createServer((req, res) => {
+    if (req.url.slice(1) === 'favicon.ico') {
+      return;
+    }
+
+    if (!req.url.includes('file')) {
+      console.log('Start request with "file"');
+      res.end();
+
+      return;
+    }
+
+    try {
+      const pathSet = new Set(req.url.split('/'));
+
+      pathSet.delete('');
+
+      const pathName = [...pathSet];
```

Comment:
Didn't notice this in prev task :sweat_smile: 
This will break paths with duplicate folders, e.g `/file/test/test/image.jpg`